releases/release-5.0.0.md

```diff
@@ -0,0 +1,468 @@
+
+In the pessimistic transaction mode, if the tables involved in a transaction contain concurrent DDL operations or `SCHEMA VERSION` changes, the system automatically updates the transaction's `SCHEMA VERSION` to the latest to ensure the successful transaction commit, and to avoid that the client receives the `Information schema is changed` error when the transaction is interrupted by DDL operations or `SCHEMA VERSION` changes.
+
+This feature is disabled by default. To enable the feature, modify the value of [`tidb_enable_amend_pessimistic_txn`](/system-variables.md#tidb_enable_amend_pessimistic_txn-new-in-v407) system variable. This feature is introduced in v4.0.7 and has the following issues fixed in v5.0:
+
++ The compatibility issue that occurs when TiDB Binlog executes `Add Column` operations
++ The data inconsistency issue that occurs when using the feature together with the unique index
++ The data inconsistency issue that occurs when using the feature together with the added index
+
+Currently, this feature still has the following incompatibility issues:
+
++ Transaction’s semantics might change when there are concurrent transactions
++ Known compatibility issue that occurs when using the feature together with TiDB Binlog
++ Incompatibility with `Change Column`
+
+### Character set and collation
+
+- Support the `utf8mb4_unicode_ci` and `utf8_unicode_ci` collations. [User document](/character-set-and-collation.md#new-framework-for-collations), [#17596](https://github.com/pingcap/tidb/issues/17596)
+- Support the case-insensitive comparison sort for collations
+
+### Security
+
+[User document](/log-redaction.md), [#18566](https://github.com/pingcap/tidb/issues/18566)
+
+To meet security compliance requirements (such as *General Data Protection Regulation*, or GDPR), the system supports desensitizing information (such as ID and credit card number) in the output error messages and logs, which can avoid leaking sensitive information.
+
+TiDB supports desensitizing the output log information. To enable this feature, use the following switches:
+
++ The global variable [`tidb_redact_log`](/system-variables.md#tidb_redact_log). Its default value is `0`, which means that desensitization is disabled. To enable desensitization for tidb-server logs, set the variable value to `1`.
++ The configuration item `security.redact-info-log`. Its default value is `false`, which means that desensitization is disabled. To enable desensitization for tikv-server logs, set the variable value to `true`.
++ The configuration item `security.redact-info-log`. Its default value is `false`, which means that desensitization is disabled. To enable desensitization for pd-server logs, set the variable value to `true`.
++ The configuration item `security.redact_info_log`. Its default value is `false`, which means that desensitization is disabled. To enable desensitization for tiflash-server and tiflash-learner logs, set the variable value to `true`.

```

Comment:
@ilovesoup Please check it is `redact-info-log` or `redact_info_log`

Reply:
Modified by commit below